modules/osdk-cli-reference-create.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-create_{context}"]
+= create
+
+The `operator-sdk create` command is used to create, or _scaffold_, a Kubernetes API.
+
+[id="osdk-cli-ref-create-api_{context}"]
+== api
+
+The `create api` subcommand scaffolds a Kubernetes API. The subcommand must be run in a project that was initialized with the `init` command.
+
+.`create api` flags
+[options="header",cols="1,3"]
+|===
+|Flag |Description
+
+|`-h`, `--help`
+|Help output for the `run bundle` subcommand.
+
+|===
+
+== webhook
+The `create webhook` subcommand scaffolds a webhook for an API resource. The subcommand must be run in a project that was initialized with the `init` command.
```

Comment:
Please add a blank line between the heading and the first paragraph.  I don't know if there is explicit guidance on this point or if it is convention, such as shown in the mod docs guidelines for [reference modules](https://redhat-documentation.github.io/modular-docs/#creating-reference-modules)